mne/evoked.py

```diff
@@ -727,6 +727,42 @@ def plot_white(self, noise_cov, show=True):
         return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
                                   rank=None, show=show)
 
+    def animate(self, ch_type, frames=5, interval=100, butterfly=False,
+                blit=True, show=True):
+        """Make animation of evoked data as topomap timeseries.
+
+        Parameters
+        ----------
+        ch_type : str
+            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
+        frames : int | list of ints
+            If int, the number of frames to animate. If list of ints, the
+            indices to plot in the animation. Defaults to 5.
+        interval : int
+            The time interval before drawing a new frame as milliseconds.
+            Defaults to 100.
```

Comment:
Well, I would think that the typical use case is that the user finds a frame rate that is pleasing to the eye and uses that for all the animations, regardless of the sfreq. To me frame rate intuitively means how often the frame in the animation refreshes. I would find it surprising to find it different for different sampling frequencies.